src/pages/settings/Security/SecuritySettingsPage.tsx

```diff
@@ -43,29 +42,43 @@ function SecuritySettingsPage() {
             onPress: item.action,
             shouldShowRightIcon: true,
             link: '',
+            wrapperStyle: [styles.sectionMenuItemTopDescription],
         }));
-    }, [translate, waitForNavigate]);
+    }, [translate, waitForNavigate, styles]);
 
     return (
-        <IllustratedHeaderPageLayout
-            title={translate('initialSettingsPage.security')}
-            onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
-            shouldShowBackButton={isSmallScreenWidth}
-            illustration={LottieAnimations.Safe}
-            backgroundColor={theme.PAGE_THEMES[SCREENS.SETTINGS.SECURITY].backgroundColor}
-            shouldShowOfflineIndicatorInWideScreen
-            icon={Illustrations.LockClosed}
+        <ScreenWrapper
             testID={SecuritySettingsPage.displayName}
+            includeSafeAreaPaddingBottom={false}
+            shouldEnablePickerAvoiding={false}
+            shouldEnableMaxHeight
+            shouldShowOfflineIndicatorInWideScreen
         >
-            <ScrollView contentContainerStyle={[styles.flexGrow1, styles.flexColumn, styles.justifyContentBetween]}>
-                <View style={[styles.flex1]}>
-                    <MenuItemList
-                        menuItems={menuItems}
-                        shouldUseSingleExecution
-                    />
+            <HeaderWithBackButton
+                title={translate('initialSettingsPage.security')}
+                shouldShowBackButton={isSmallScreenWidth}
+                onBackButtonPress={() => Navigation.goBack()}
```

Comment:
```suggestion
                onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
```

Reply:
It was changed on main as well:
https://github.com/Expensify/App/commit/03896dc5897d256d8842e0f1d0bd692c70944a05